docs/sematic-ui.md

```diff
@@ -68,6 +68,9 @@ Each box with a status pin (the tiny green spots) corresponds to a Sematic
 Function's run. You can click on them to open the corresponding run in the
 central [Run Panel](#run-panel) in order to inspect its details.
 
+You can zoom in and out on the graph diagram by using the trackpad, or with
+`CMD+scroll`.
```

Comment:
I presume on Linux/Windows it's `Ctrl+scroll`?

Reply:
Did a websearch, and apparently this is correct. I have no way of testing this. Updated.